magento/magento2
# Issue 19456

### Description (*)


### Fixed Issues (if relevant)

1. magento/magento2#19456 : Admin Sub-Menu Navigation Feature

### Manual testing scenarios (*)

1. Go to admin > you can see the submenu have icons that will indicate the sub-menu

### Contribution checklist (*)
